model_prices_and_context_window.json

```diff
@@ -2769,6 +2769,42 @@
         "supports_response_schema": true,
         "source": "https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#gemini-1.5-flash"
     },
+    "gemini-2.0-flash-exp": {
```

Comment:
both of these have the same key - this will lead to an error. 

I believe for google ai studio the norm is `gemini/<model-name>`